charts/traffic/templates/service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}
+spec:
+  selector:
+    app.kubernetes.io/name: {{ template "iter8-traffic.name" . }}
+  ports:
+  - name: grpc
+    port: {{ .Values.abn.port }}
+    targetPort: {{ .Values.abn.port }}
+  - name: http
```

Comment:
I wonder if we should consider more descriptive names just to be more clear

Reply:
unfortunately, kubernetes likes these names.